src/dashboard/apps/consent/tests/test_signals.py

```diff
@@ -0,0 +1,24 @@
+"""Dashboard consent signals tests."""
+
+import pytest
+
+from apps.consent.models import Consent
+from apps.consent.utils import consent_end_date
+from apps.core.factories import DeliveryPointFactory
+
+
+@pytest.mark.django_db
+def test_signal_triggered_on_delivery_point_creation():
+    """Tests if the signal is triggered when creating a DeliveryPoint."""
+    assert Consent.objects.count() == 0
+
+    # Create new delivery point
+    delivery_point = DeliveryPointFactory()
+
+    # check if consent was created with signals after delivery_point creation.
+    assert Consent.objects.count() == 1
+
+    consent = Consent.objects.first()
+    assert consent.delivery_point == delivery_point
+    expected_date = consent_end_date()
+    assert consent.end.replace(microsecond=0) == expected_date.replace(microsecond=0)
```

Comment:
This can lead to flaky tests, you should instead monkeypatch `datetime.now` or use [freezegun](https://github.com/spulec/freezegun) or alike.